Week1/week1-tasks/src/ArrayFill_II.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Array fill II problem
+ * @version 28.3.2023
+ * @author Karim Adel
+ */
+import java.util.Scanner;
+
+public class ArrayFill_II {
+    public static void main(String[] args) {
+        /* New Scanner instance to read input from user*/
+        Scanner in = new Scanner(System.in);
+
+        /* Read T */
+        int T = in.nextInt();
+
+        /* Counter */
+        int cnt = 0;
```

Comment:
Use meaningful variable names: Variable names should be descriptive and convey the purpose of the variable.

use counter instead of cnt